pages/common/coffee.md

```diff
@@ -0,0 +1,23 @@
+# coffee
+
+> Executes CoffeeScript scripts or compiles them into JavaScript.
+
+- Run a script:
+
+`coffee {{path/to/filename}}.coffee`
```

Comment:
Use {{path/to/file}} everywhere. I think that is more idiomatic with the rest of the pages.